tests/fields/fields.go

```diff
@@ -65,6 +65,7 @@ func main() {
 		{"orgs/google-test", &github.Organization{}},
 		{"repos/google/go-github", &github.Repository{}},
 		{"/gists/9257657", &github.Gist{}},
+		{"repos/google/go-github/issues/1", &github.Issue{}},
```

Comment:
You can move it up by one line. That way it'll be grouped with the similar `{"repos/google/go-github", &github.Repository{}},` entry.